src/components/language_picker.html

```diff
@@ -0,0 +1,39 @@
+{% import "Macros" as macros %}
```

Comment:
Macros work (import the name defined in the `book.toml` and not the file name. You could name it with the file name too)

Reply:
I see! This is the difference in thinking: I was expecting the user to declare an include directory and then rely solely on the mechanisms in the template engine (import a file by it's relative path, for example).

I believe you've instead built more infrastructure to make it easier to use — but the problem with that is that the user now has to understand the custom logic here (and you and I have to document and maintain this).

Reply:
I want to support both options, but to do use a macro like this in `index.hbs`, you will need to write sometimes 4 `{` in a row `{{{{ attribute | get(key="style") }}}}`, I find that very clunky, when using HTML syntax is pretty easy to understand for people already writing HTML.

Reply:
You might even have to write 6 in a row when I think about it

Reply:
> I want to support both options

Please don't: avoid building up your own vocabulary here since nobody but you will know about it :slightly_smiling_face: I would like us to have as little code to maintain as possible here.